server/api/models/question.js

```diff
@@ -10,6 +11,7 @@ const QuestionSchema = mongoose.Schema({
   options: [{ type: String }]
```

Comment:
Shouldn't the schema be like:
questions: [
{
id,
options : [],
correctAnswer
}
]